components/util.py

```diff
@@ -176,3 +177,52 @@ async def admin_check(chat_data: CD, chat: Chat, who_banned: User) -> bool:
     if who_banned not in [admin.user for admin in admins]:
         return False
     return True
+
+
+async def get_bot_from_token(token: str) -> Optional[User]:
+    bot = Bot(token)
+
+    try:
+        user = await bot.get_me()
+        return user
+
+    # raised when the token isn't valid
+    except TelegramError:
```

Comment:
```suggestion
    except InvalidToken:
```

e.g. a timeout error shouldn.t be treated as invalid token :)

Reply:
I just wrote that based on the docs, should have checked what the exception was, tho.
Maybe this should be updated
![image](https://user-images.githubusercontent.com/58857054/210065252-e9c7dd2a-564a-43c2-96fe-c94caea1cc2e.png)